engine/src/main/java/org/terasology/logic/inventory/ItemComponent.java

```diff
@@ -80,6 +80,10 @@
      */
     public int baseDamage = 1;
 
+    public boolean pickableUp = true;
```

Comment:
Could you create another component that has this delayed pickup stuff on it?  I have dreams to slim ItemComponent down until multiple components so that the item system becomes more modular and less monolithic.